inbox/models/message.py

```diff
@@ -140,16 +140,26 @@ def categories_changes(self, has_changes):
     # The uid as set in the X-INBOX-ID header of a sent message we create
     inbox_uid = Column(String(64), nullable=True, index=True)
```

Comment:
I think you can actually directly rename this column to `nylas_uid` if you specify the `name` parameter to Column: http://docs.sqlalchemy.org/en/latest/core/metadata.html#sqlalchemy.schema.Column.__init__

If this works, it'd prevent people from accidentally using the old name on the model in the future!